glomentorprogram/src/stories/SearchForm.stories.js

```diff
@@ -0,0 +1,20 @@
+import SearchForm from '../components/SearchForm/SearchForm';
+
+export default {
+	title: 'SearchForm',
+	component: SearchForm,
+	decorators: [(story) => <div style={{ padding: '3rem' }}>{story()}</div>],
+	argTypes: {
+		handleClick: {
```

Comment:
@bcvitanusic so following the approaches of using `argTypes annotation` approach, you should use the name prop's that will fire the action. In case of SearchForm component, you should call the onSearch prop. So just update the annotation to something like:

```
...
 argTypes: {
   onSearch: { ... }
 }
...
```